src/qutip_cupy/__init__.py

```diff
@@ -5,6 +5,13 @@
 # to an official optional dependency
 import warnings
 
+try:
+    import cupy
+except ModuleNotFoundError:
+    raise RuntimeError('''qutip_cupy requires cupy to be installed, please install
+                         cupy by following the instructions at https://docs.cupy.dev/en/stable/install.html''')
+del cupy
```

Comment:
If you like, you can also put the `del cupy` inside the `try: ...` block.

Reply:
(Just so it is visibly closer to the import line and one doesn't have to look further down the page).